integration_tests/onedut_oneotg_tests/aggregate/aggregate_test.go

```diff
@@ -0,0 +1,504 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rt_5_2_aggregate_test
```

Comment:
Is this test the same as the one in featureprofiles?

Reply:
with some minor changes, yes